core/src/header.rs

```diff
@@ -483,6 +483,14 @@ where
     ) -> Result<Vec<DicomDateTime>, ConvertValueError> {
         self.value().to_multi_datetime(default_offset)
     }
+
+    /// Retrieve items stored in value
+    ///
+    /// If value is not a sequence return None.
+    ///
+    pub fn to_items(&self) -> Option<&[I]> {
```

Comment:
As per the [C-CONV](https://rust-lang.github.io/api-guidelines/naming.html#c-conv) guideline. This should rather be named `items` (named as a getter) or `as_items` (thus treating as a borrowed view over the value). Otherwise, one could expect `to_items` to return an independent list of items, which is not intended here.

```suggestion
    pub fn to_items(&self) -> Option<&[I]> {
```